src/c++/perf_analyzer/genai-pa/genai_pa/llm_profile.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+# Copyright 2023-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met: * Redistributions of source code must retain the above copyright
```

Comment:
2024 only? Also this seems corrupted and is missing an endline?

Reply:
documented. Resolving on behalf of @tgerdesnv